UI/CUIWnd.cpp

```diff
@@ -616,6 +621,67 @@ void CUIWnd::ResetDefaultPosition() {
 GG::Rect CUIWnd::CalculatePosition() const
 { return GG::Rect(INVALID_X, INVALID_Y, INVALID_X, INVALID_Y); }
 
+void CUIWnd::SetDefaultedOptions() {
+    std::unordered_set<std::string> retval;
```

Comment:
Remove `retval` it is unused.

Reply:
@LGM-Doyle You've done a bunch of work on Wnd setup and related stuff, so I thought you might have some perspective on the timing of option default setting and layout happening...

Reply:
It looks fine .  If there are non-obvious bugs they will be fixed in due course.

Reply:
Removed, ty for reviewing